### PR TITLE
Use 'missing_required_lib' to report import errors

### LINKED ISSUE #326: There is a convention for importing libraries, we should conform to.
We should conform with:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_best_practices.html#importing-and-using-shared-code

### PR DESCRIPTION
**Pull request by mdellweg**
Fixes #326

### CODE REVIEW

File: plugins/module_utils/foreman_helper.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import yaml
 import traceback
 
-from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
```
Comment (evgeni) [inline plugins/module_utils/foreman_helper.py]: This is Ansible 2.8+ only :/

**Comment by evgeni** [comment]
marked this as WIP until 2.8 is "old enough"

**Comment by mdellweg** [comment]
"Wait In Progress"
I was about to create a generic "blocked" label, but i think we have nough of em.

**Comment by evgeni** [comment]
a rebased version is in #839

**Pull request closed without merging**